jobserver/settings.py

```diff
@@ -259,6 +262,41 @@
 X_FRAME_OPTIONS = "SAMEORIGIN"
 
 
+# CSP
+# https://django-csp.readthedocs.io/en/latest/configuration.html
+CSP_REPORT_ONLY = DEBUG
+CSP_DEFAULT_SRC = ["'none'"]
+CSP_CONNECT_SRC = ["'self'", "https://plausible.io"]
+CSP_FONT_SRC = ["'self'"]
+CSP_IMG_SRC = [
+    "'self'",
+    "blob:",
+    "data: w3.org/svg/2000",
+    "https://github.com",
+    "https://avatars.githubusercontent.com",
+]
+CSP_MANIFEST_SRC = ["'self'"]
+
+# Duplicate the *_ELEM settings for Firefox
+# https://bugzilla.mozilla.org/show_bug.cgi?id=1529338
+CSP_SCRIPT_SRC = CSP_SCRIPT_SRC_ELEM = ["'self'", "https://plausible.io"]
+CSP_STYLE_SRC = CSP_STYLE_SRC_ELEM = ["'self'"]
+
+# which directives to set a nonce for
+CSP_INCLUDE_NONCE_IN = ["script-src-elem"]
```

Comment:
Does "script-src" also need to be in this list?

Reply:
It does indeed, fixed ✅